system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_ethereum_config.rs

```diff
@@ -25,3 +38,183 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 	snowbridge_pallet_outbound_queue::Pallet<Runtime>,
 	snowbridge_core::AgentIdOf,
 >;
+
+parameter_types! {
+	// The gateway address is set by governance.
+	pub storage EthereumGatewayAddress: H160 = H160::zero();
+}
+
+impl snowbridge_pallet_inbound_queue::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Verifier = snowbridge_pallet_ethereum_client::Pallet<Runtime>;
+	type Token = Balances;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type XcmSender = xcm_config::XcmRouter;
+	#[cfg(feature = "runtime-benchmarks")]
+	type XcmSender = benchmark_helpers::DoNothingRouter;
+	type ChannelLookup = EthereumSystem;
+	type GatewayAddress = EthereumGatewayAddress;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Helper = Runtime;
+	type MessageConverter = MessageToXcm<
+		CreateAssetCall,
+		bp_asset_hub_kusama::CreateForeignAssetDeposit,
+		InboundQueuePalletInstance,
+		AccountId,
+		Balance,
+	>;
+	type WeightToFee = WeightToFee;
+	type LengthToFee = ConstantMultiplier<Balance, TransactionByteFee>;
+	type MaxMessageSize = ConstU32<2048>;
+	type WeightInfo = crate::weights::snowbridge_pallet_inbound_queue::WeightInfo<Runtime>;
+	type PricingParameters = EthereumSystem;
+	type AssetTransactor = <xcm_config::XcmConfig as xcm_executor::Config>::AssetTransactor;
+}
+
+impl snowbridge_pallet_outbound_queue::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Hashing = Keccak256;
+	type MessageQueue = MessageQueue;
+	type Decimals = ConstU8<12>;
+	type MaxMessagePayloadSize = ConstU32<2048>;
```

Comment:
If this has to be the same as the `MaxMessageSize`, maybe a constant would be good?

Reply:
Definitely agree that the magic numbers needs to be hoisted into documented constants. Though `MaxMessagePayloadSize` does have a different purpose to `snowbridge_pallet_inbound_queue::MaxMessageSize`.

* inbound_queue::MaxMessageSize: Worst case message size (including proofs), used to calculate refunds for relayers.
* outbound_queue::MaxMessagePayloadSize: Maximum allowed size of encoded message payload that is committed for delivery to Ethereum